packages/main/cypress/specs/Popover.cy.tsx

```diff
@@ -3,39 +3,137 @@ import Toolbar from "../../src/Toolbar.js";
 import Button from "../../src/Button.js";
 import Label from "../../src/Label.js";
+import List from "../../src/List.js";
+import ListItem from "../../src/ListItemStandard.js";
+import Input from "../../src/Input.js";
 
 describe("Rendering", () => {
 	it("tests arrow positioning", () => {
 		cy.mount(
 			<>
 				<div id="icon1" tabindex="0" style="width: 10px; height: 10px; background:red;"></div>
 				<Popover id="popup"
-						 opener="icon1"
-						 headerText="Newsletter subscription"
-						 placement="End"
-						 verticalAlign="Top">
+					opener="icon1"
+					headerText="Newsletter subscription"
+					placement="End"
+					verticalAlign="Top">
 					<Button id="btnClosePopover">Close</Button>
 				</Popover>
 			</>
 		);
 
-		// act
-		cy.get("#popup")
+		cy.get("[ui5-popover]")
 			.invoke("prop", "open", "true");
 
-		cy.get("#popup")
+		cy.get<Popover>("[ui5-popover]")
 			.ui5PopoverOpened();
 
-		cy.get("#popup")
+		cy.get("[ui5-popover]")
 			.shadow()
 			.find(".ui5-popover-arrow")
 			.should("be.visible");
 
-		cy.get("#popup")
+		cy.get("[ui5-popover]")
 			.shadow()
 			.find(".ui5-popover-arrow")
 			.should("have.css", "transform", "matrix(1, 0, 0, 1, 0, -42)");
 	});
+
+	it("tests rendering of the popover when open attribute is set", () => {
+		cy.mount(
+			<>
+				<Button id="popoverOpen">Open</Button>
+				<Popover id="popover" opener="popoverOpen" open={true}>
+					<Button>Close</Button>
+				</Popover>
+			</>
+		);
+
+		cy.get("[ui5-popover]").should("have.attr", "open");
+	});
+
+	it("test initial focus", () => {
+		cy.mount(
+			<>
+				<Button id="btnOpen">Open</Button>
+				<Popover id="popover" opener="btnOpen" open={true}>
+					<Button id="popoverBtn">Button</Button>
+				</Popover>
+			</>
+		);
+
+		cy.get("#popoverBtn").should("be.focused");
+	});
+
+	it("test _open", () => {
+		cy.mount(
+			<>
+				<Button id="popoverOpen">Open</Button>
+				<Popover id="popover1" opener="popoverOpen" open={true}>
+					<div id="popoverCont1">Content</div>
+				</Popover>
+			</>
+		);
+
+		cy.get("#popoverOpen").should("exist");
+		cy.get("#popover1").should("have.attr", "open");
+		cy.get("#popover1").should("have.prop", "_opened");
+
+		cy.get("#popoverCont1").should("exist");
+	});
+
+	it("Header text attribute is propagated", () => {
+		cy.mount(
+			<>
+				<Popover id="pop" headerText="Initial text">
+					<div>Content</div>
+				</Popover>
+			</>
+		);
+
+		cy.get("[ui5-popover]")
+			.shadow()
+			.find(".ui5-popup-header-text")
+			.should("have.text", "Initial text");
+	});
+
+	it("Popover arrow", () => {
+		cy.mount(
+			<>
+				<Button id="btn">Open</Button>
+				<Popover id="pop" opener="btn" headerText="Popover Header" open>
+					<div>Content</div>
+				</Popover>
+			</>
+		);
+
+		cy.get("[ui5-button]").realClick();
+
+		cy.get("[ui5-popover]")
+			.shadow()
+			.find(".ui5-popover-arrow")
+			.should("be.visible");
+	});
+
+	it("Popover arrow on hide-arrow", () => {
```

Comment:
first assert that the arrow is shown, then toggle it and then assert that it is hidden